filters/filtercontainer.h

```diff
@@ -30,9 +30,15 @@ class FilterContainer {
     virtual void onFilterRemoved(Filter* filter) = 0;
     virtual void onFiltersCleared() = 0;
 
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```

Comment:
Both Qt 5 and 6 have a `qsizetype`

Reply:
That's true, but the problem is in differences regarding qml list properties:

https://codebrowser.dev/qt5/qtdeclarative/src/qml/qml/qqmllist.h.html#QQmlListProperty::CountFunction
https://codebrowser.dev/qt6/qtdeclarative/src/qml/qml/qqmllist.h.html#QQmlListProperty::CountFunction

with `using sizetype = qsizetype;` wouldn't compile on Qt 5.